File: code/Go/salario.go
package main

import "fmt"

func main() {
	var nome string = ""
	var salario float64 = 0
	fmt.Println("Nome do Funcionário:")
	fmt.Scanf("%s", &nome)
	fmt.Println("Salário:")
	fmt.Scanf("%f", &salario)

	fmt.Printf("O funcionário %s tem um salário de R$%.2f em abril\n", nome, salario)
}

File: code/C/salario.c
#include <stdio.h>

int main(void) {
    // Declaração de variáveis.
    char nome[61]; // vetor de caracteres
    float salario = 0;
    printf("Nome do funcionário:");
    scanf("%s", &*nome);

    printf("Salário:");
    scanf("%f", &salario);

    printf("O funcionario %s tem um salário de R$%.2f em abril\n", nome, salario);

}